Jenkinsfile

```diff
@@ -48,40 +41,24 @@ pipeline {
             // so we can retrieve the version in later steps
             sh "echo \$(jx-release-version) > VERSION"
             sh "mvn versions:set -DnewVersion=\$(cat VERSION)"
+            sh "git add --all"
+            sh "git commit -m 'release \$(cat VERSION)' --allow-empty"
```

Comment:
Realy small detail, but can we use the first capital for the commit message?  
`sh "git commit -m 'Release \$(cat VERSION)' --allow-empty"`